ktor-io/jvmAndPosix/test/ByteWriteChannelSinkTest.kt

```diff
@@ -0,0 +1,26 @@
+import io.ktor.utils.io.*
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.test.runTest
+import kotlinx.io.buffered
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+/*
+ * Copyright 2014-2025 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
```

Comment:
Nit. Could you move copyright to the top of a file in the all new test files?

Reply:
sure